source/macos/AdaptiveCards/AdaptiveCardsTests/Renderers/InputTimeRendererTests.swift

```diff
@@ -19,7 +19,7 @@ class InputTimeRendererTest: XCTestCase {
         inputTime = .make(value: val)
 
         let inputTimeField = renderTimeInput()
-        XCTAssertEqual(inputTimeField.dateValue, val)
+        XCTAssertEqual(inputTimeField.dateValue, "15:30")
```

Comment:
Change the value in line number 18 to `let val: String = "15:30"` dont hardcode here.